scripts/install

```diff
@@ -167,7 +200,17 @@ ynh_use_logrotate
 #=================================================
 ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
 
-yunohost service add $app --description="Jellyfin media center" --log_type="systemd"
+# Build the ports list
+needed_ports=()
+(( $discovery_server == 1 )) && needed_ports+=( "$discovery_server_port" )
```

Comment:
```suggestion
(( $discovery_service == 1 )) && needed_ports+=( "$discovery_service_port" )
```

Reply:
aaaaaand I was like "noice, `server` and `client` have the same length".